client/src/Movies/MovieList.js

```diff
@@ -11,44 +13,22 @@ export default class MovieList extends Component {
 
   componentDidMount() {
     axios
-      .get('http://localhost:5000/api/movies')
+      .get("http://localhost:5000/api/movies")
       .then(response => {
         this.setState(() => ({ movies: response.data }));
       })
       .catch(error => {
-        console.error('Server Error', error);
+        console.error("Server Error", error);
       });
   }
 
   render() {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <MovieCard movie={movie} key={movie.id} addToSavedList={this.props.addToSavedList} />
```

Comment:
this probably doesn't work, because inside of MovieCard `props.saveMovie` is invoked but you don't provide that prop here. Suggest:
```suggestion
          <MovieCard movie={movie} key={movie.id} saveMovie={this.props.addToSavedList} />
```